source/jormungandr/jormungandr/autocomplete/kraken.py

```diff
@@ -92,7 +92,7 @@ def geo_status(self, instance):
         return status
 
     @get_serializer(serpy=api.PlacesSerializer)
-    def get_by_uri(self, uri, request_id, instances=None, current_datetime=None):
+    def get_by_uri(self, uri, request_id, instances=None, current_datetime=None, _add_poi_shape=False):
```

Comment:
If I understand, we don't use the parameter _add_poi_shape in kraken.

Reply:
:+1: